charset_normalizer/utils.py

```diff
@@ -73,6 +73,7 @@ def is_latin(character: str) -> bool:
     return "LATIN" in description
 
 
+@lru_cache(maxsize=UTF8_MAXIMAL_ALLOCATION)
```

Comment:
Let's not increase RAM usage when it's not really necessary.

Reply:
without lru_cache ([cachegrind.out.original.zip](https://github.com/Ousret/charset_normalizer/files/8598533/cachegrind.out.original.zip))

![without lru_cache](https://user-images.githubusercontent.com/65482418/166166705-230d74cb-5a40-4f00-b579-bd2464d23dc9.jpg)
---
with lru_cache ([cachegrind.out.commits.zip](https://github.com/Ousret/charset_normalizer/files/8598511/cachegrind.out.commits.zip))

![with lru_cache](https://user-images.githubusercontent.com/65482418/166166717-ef9a9140-912e-44be-a020-124016e7dee9.jpg)
---